cmd/configure_test.go

```diff
@@ -7,76 +7,86 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type testDefinition struct {
```

Comment:
What do you think about calling this type `testCase` ? testDefinition is a bit vague